module-1/desgindoc.md

```diff
@@ -0,0 +1,89 @@
+# Overview
+Implement For You recommendation system, that will provide personal goods recomendations based on user preferences inside Goodsend app.
+
+Goodsend is a **social wishlist app** that lets users save products with a simple tap, organize them into collections or collages, share with friends, and explore products shared by others.
+
+DAU ~200 users
+
+Images of product:
+User profile          |     Explore (chronological) page |     Good(item) modal
+:-------------------------:|:-------------------------:|:-------------------------:
+![alt text](image.png)  |  ![alt text](image-1.png) |  ![alt text](image-2.png)
+
+
+
+# Motivation
+Currently, users can explore new goods from friends feed, chronological feed or one for all users curated weekly feed.
+
+**Hypothesis:** If we implement For You feed with personalized recommendations we will improve user experience in exploring new items, increase engagement, retention rate and session time.
+
+**Success metrics:**
+Increased engagement (DAU, session time, retention rate)
+
+# Requirements & Constraints
+Should work only on users with sufficient data(>10 goods, tbd). Cold-start problem is out of scope.
```

Comment:
why 10? is it a guess or is something else behind it?

Reply:
more of a guess